### ISSUE TITLE
Web response slow. 

### ISSUE THREAD

**Issue by Clanggedin** [jasoncoon/esp8266-fastled-webserver#84]
Last year I used this code on a Daft Punk helmet I built and it worked well on a WeMos D1 Mini. I end blowing the board because I got the LEDs bright on accident. I have since then replaced the board, but the response time of the site is super slow if it ever even loads. If I cancel the request then I get a very basic page that has a few links which one is to a "Simple" page that only has an on-off.  I am not sure what has changed to make the response time super slow. Changing devices that I use to connect to the D1 Mini does not resolve the issue. I have alway connected in the past in AP mode without much issues. Not sure where to look to resolve this issue. Wiping the board and reintstalling everything did not resolve the issue.

**Comment by tobi01001** [comment]
Does is somehow load the js files from the web?
Then it will time out as in ap mode there is no route to the web. In this
case you could upload the js files to the esp and load them from there.

Clanggedin <notifications@github.com> schrieb am Di., 7. Aug. 2018, 16:23:

**Comment by jasoncoon** [comment]
Yeah, I think there are .js files that are not yet included in the web app files uploaded to SPIFFS.  I hope to investigate this and other issues soon.
